File: new/atoms/terminal-outcomes.test.mjs
import * as m from './terminal-outcomes.mjs';
const SNAP = {"TERMINAL_OUTCOMES":"[\"donated\",\"refused\",\"callback\",\"done\"]"};
let f = 0;
for (const [k, s] of Object.entries(SNAP)) if (JSON.stringify(m[k]) !== s) { console.error('✗ ' + k + ' סטה מהצילום'); f = 1; }
if (f) process.exit(1); console.log('✓ terminal-outcomes: צילום-ערך תואם — ירוק');

File: new/dart-maor/terminal-outcomes_test.dart
// בדיקת-חוזה (רתמת-זהב) · terminal-outcomes — צילום-ערך jsonEncode≡JSON.stringify.
// חוללה מכנית (gen-data-dart). הרצה: dart run --enable-asserts ⇒ OK
import 'dart:convert';
import 'terminal-outcomes.dart';

void main() {
  // ‏1) צילום-ערך: jsonEncode ≡ JSON.stringify של המקור (זהה-ביט).
  if (jsonEncode(terminalOutcomes) != '["donated","refused","callback","done"]') {
    throw StateError('FAIL terminalOutcomes: צילום-הערך סטה');
  }
  print('OK terminal-outcomes: 1 data-snapshot(s) passed');
}
